aries_vcx/src/protocols/issuance_v2/holder/states/complete.rs

```diff
@@ -0,0 +1,23 @@
+use std::marker::PhantomData;
+
+use messages::msg_fields::protocols::cred_issuance::v2::ack::AckCredentialV2;
+
+use crate::protocols::issuance_v2::formats::holder::HolderCredentialIssuanceFormat;
+
+pub struct Complete<T: HolderCredentialIssuanceFormat> {
+    pub(crate) ack: Option<AckCredentialV2>,
```

Comment:
This should not be here. The ack is generated and sent, it's not useful to track how exactly the `Ack` message we have sent looked like.